pom.xml

```diff
@@ -34,27 +34,21 @@
   </repositories>
   <dependencies>
     <dependency>
-      <groupId>com.sk89q</groupId>
-      <artifactId>worldedit</artifactId>
-      <version>5.5.9-SNAPSHOT</version>
+      <groupId>com.sk89q.worldedit</groupId>

```

Comment:
It probably makes sense to eventually move this to Gradle to be the same as the other EH projects